Doc/whatsnew/3.10.rst

```diff
@@ -2152,8 +2152,7 @@ Porting to Python 3.10
 * The ``PY_SSIZE_T_CLEAN`` macro must now be defined to use
   :c:func:`PyArg_ParseTuple` and :c:func:`Py_BuildValue` formats which use
   ``#``: ``es#``, ``et#``, ``s#``, ``u#``, ``y#``, ``z#``, ``U#`` and ``Z#``.
-  See :ref:`Parsing arguments and building values
-  <arg-parsing>` and the :pep:`353`.
+  See :ref:`arg-parsing` and the :pep:`353`.
```

Comment:
```suggestion
  See :ref:`arg-parsing` and :pep:`353`.
```

If we're changing this line anyway, the definite article is better left out here